### ISSUE TITLE
The finetuning results on Bridge are not good.

### ISSUE THREAD

**Issue by swstbecrpgmail** [openvla/openvla#262]
Thank you very much for open-sourcing this outstanding work!

I'm trying to finetune OpenVLA on the Bridge-OXE dataset with 3xRTX5880ada, but my loss curve fluctuates violently, and the model isn't fitting well. Are there any methods to improve the finetuning results?

Here's my loss curve(smoothing=0.99), action fitting plots and training parameters.

![Image](https://github.com/user-attachments/assets/3ed80034-2564-4c34-badd-efbb6ac28296)

![Image](https://github.com/user-attachments/assets/17abe2d7-317e-4885-b2b0-a9c2247f8559)

![Image](https://github.com/user-attachments/assets/bc7a0dd4-539a-4439-91e0-baae691a1b36)

Additionally, I compute the ground-truth actions following bridge_oxe_dataset_transform() function.

![Image](https://github.com/user-attachments/assets/73036f84-e7c0-40e0-bbe8-5fed488472b5)

If anyone has any possible suggestions for improvement, such as adjusting parameters or pointing out potential mistakes, feel free to leave comments! Thank you very much!
